whitepaper/whitepaper.md

```diff
@@ -706,19 +706,25 @@ Once an API call to Hyperledger Cactus framework is requested by a User, Busines
 Following diagram shows the architecture of Hyperledger Cactus based on the discussion made at Hyperledger Cactus project calls.
 The overall architecture is as the following figure.
 
-<img src="./architecture-with-plugin-and-routing.png" width="700">
+<img src="./cactus_arch.svg" width="700">
 
 ### 5.2.1 Definition of key components in system architecture
 
 Key components are defined as follows:
-- **Application user**: The entity submits API calls to "Cactus Routing Interface". Note: this component is exist outside of Cactus service system.
+- ~~**Application user**: The entity submits API calls to "Cactus Routing Interface". Note: this component is exist outside of Cactus service system.~~
 - **Business Logic Plugin**: The entity executes business logic and provide integration services that are connected with multiple blockchains. The entity is composed by web application or smart contract on a blockchain. The entity is a single plugin and required for executing Hyperledger Cactus applications.
+- **Command API Server (cmd-api-server)**: The server accepts a request from an End-user Application, and return a response depending on the status of the targeted trade. Trade ID will be assigned when a new trade is accepted.
+- **End-user Application**: The entity submits API calls to request a trade, which invokes a set of transactions on Ledger by the Business Logic Plugin.
```

Comment:
> The entity submits API calls to request a trade, which invokes a set of transactions on Ledger by the Business Logic Plugin.

@sfuji822 Would it make sense to mention something more generic than requesting a `trade`? For example the end user application could submit an API call to request information (e.g. querying ledger state without modifying it).

Reply:
My intention was Trade is changing Ledger behind LedgerPlugin, but queries will not change the state of Ledger.
Do you want to have the capability to make a query through BLP? > @petermetz

Reply:
> My intention was Trade is changing Ledger behind LedgerPlugin, but queries will not change the state of Ledger.

@sfuji822 Makes sense. Do you feel that the word `trade` can really encompasses all of our non-financial use cases? As-in, would you describe an end user application uploading cat pictures to a ledger as a `trade`?

> Do you want to have the capability to make a query through BLP? > @petermetz

@sfuji822 I have no strong opinions on that. Should be up to the author of the BLP IMO.